recipe/meta.yaml

```diff
@@ -1,47 +1,55 @@
+{% set version = "2.2.21" %}
+
 package:
-    name: udunits2
-    version: 2.2.20
+  name: udunits2
+  version: {{ version }}
 
 source:
-    fn: v2.2.20.tar.gz
-    url: https://github.com/Unidata/UDUNITS-2/archive/v2.2.20.tar.gz
-    sha1: a0d30024af16749a4a5ca60133b32735ae52f9e2
+  fn: v{{ version }}.tar.gz
```

Comment:
I was always suspicious that filename element was surplus to requirements here! But I get your point about not starting filenames with numbers...